packages/dev/core/src/DeviceInput/InputDevices/deviceSource.ts

```diff
@@ -7,11 +7,13 @@ import { IKeyboardEvent, IPointerEvent, IWheelEvent } from "../../Events/deviceI
 /**
  * Subset of DeviceInput that only handles pointers and keyboard
  */
-export type DeviceSourceEvent<T extends DeviceType> =
-    T extends DeviceType.Keyboard ? IKeyboardEvent :
-    T extends DeviceType.Mouse ? IWheelEvent | IPointerEvent :
-    T extends DeviceType.Touch ? IPointerEvent :
-    never;
+export type DeviceSourceEvent<T extends DeviceType> = T extends DeviceType.Keyboard
+    ? IKeyboardEvent
+    : T extends DeviceType.Mouse
+    ? IWheelEvent | IPointerEvent
+    : T extends DeviceType.Touch
+    ? IPointerEvent
+    : never;

```

Comment:
nit: This formatting is hard to read. Did autoformatting do this?

Reply:
I won't clean this up in this PR but I'll handle it once everything's been ironed out with the new build process.